pkg/rthooks/dummy.go

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package rthooks
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cilium/tetragon/api/v1/tetragon"
+)
+
+type DummyHookRunner struct {
+	*testing.T
```

Comment:
Why do we need that here? Why not use an empty struct?

Reply:
Good point. I guess it was some leftover code. Removed it.